website/src/pages/home/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../../components/Layout'
+import { IntegrationsSection } from '../../components/IntegrationsSection'
+import { SelfHostedSection } from '../../components/SelfHostedSection'
+
+import Hero from './_Hero'
+import UseCases from './_UseCases'
+import Features from './_Features'
+
+const Home: FunctionComponent<PageProps> = props => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Sourcegraph | Code Intelligence Platform',
+            description:
+                'Address security risks, onboard to a new codebase, identify the root cause of incidents, promote code reuse, improve code health, and more with Soucegraph.',
+        }}
```

Comment:
I noticed that the [meta doc](https://docs.google.com/spreadsheets/d/1GoP8FcgFvqbxnGkaddgOVqgDAc2VDW1wCKySVChuej4/edit#gid=1572625403) asks that we include the "Sourcegraph OG" image. Should we include that here?

Reply:
Awesome! Good eye! Thanks for catching that! Looks like this was added after I last checked.

@fabicastp and I were chatting about this recently and we'll eventually be doing a big swap throughout the site for all our meta images. Exciting! ✨

Reply:
Just chatted again with @fabicastp. I'm going to replace these all now throughout the codebase while I'm at it. 🚀

Reply:
That sounds great!

Reply:
On second thought, I'm going to follow up in another PR to introduce another global change like this. I don't want to make this PR too much bigger than it already is.